.travis.yml

```diff
@@ -5,7 +5,7 @@ jdk:
 stages:
   - name: test
   - name: release
-    if: (branch = master) AND (type = push OR tag IS present)
+    if: (branch = master) AND (type = push) OR (tag IS present)
```

Comment:
Probably unnecessary, but I would group master and push conditions

Reply:
https://github.com/scalameta/metals/pull/245#discussion_r179508207